avaframe/data/avaDamBreak/Inputs/damBreak_com1DFACfg.ini

```diff
@@ -79,6 +79,7 @@ stopCrit = 0.01
 
 # friction type (SamosAT = 1, Coulomb = 2)
 frictType = 2
+frictModels = samosAT_Coulomb
```

Comment:
Maybe an explanation is needed here  for the difference between `frictType` and `frictModels`...

Reply:
good point, the frictModels is just needed for the report to put the name of the model, maybe it would make sense to use the model name instead of 1 and 2 then we would not need fricType anymore?

Reply:
Yes, I think that's a good way to solve it.